workflow-controller/controller-service/src/main/java/io/hyscale/controller/model/HyscaleCommandSpec.java

```diff
@@ -19,7 +19,7 @@
 import java.util.ArrayList;
 import java.util.List;
 
-public class DeployCommandSpec {
+public class HyscaleCommandSpec {
```

Comment:
Why HyscaleCommandSpec?
Even hyscale get service status would be a hyscale command right ?
It should be close to manifest gen & deployer . Try to think of a better name

Reply:
Since in manifest and deployer we are doing input processing. Would HyscaleInputProcessingCommandSpec be better?